src/mpid/ch4/src/ch4_spawn.h

```diff
@@ -13,6 +13,26 @@
 
 #include "ch4_impl.h"
 
+/*
+=== BEGIN_MPI_T_CVAR_INFO_BLOCK ===
+
+cvars:
+    - name        : MPIR_CVAR_CH4_COMM_CONNECT_TIMEOUT
+      category    : CH4
+      type        : int
+      default     : 180
+      class       : none
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_GROUP_EQ
+      description : >-
+        The default time out period in seconds for a connection attempt to the
+        server communicator where the named port exists but no pending accept.
+        User can change the value for a specified connection through its info
+        argument.
```

Comment:
Should we just move the existing timeout CVAR up to not have a CH3/CH4 specific name?